### ISSUE TITLE
Calendar Captcha can't be completed on Mobile

### ISSUE THREAD

**Issue by rastacalavera** [JupiterBroadcasting/jupiterbroadcasting.com#492]
### On what type of device(s) did you see this bug?

Phone

### On which browser(s) are you seeing the problem?

Safari

### Summary

When I was on guest wifi and tried to view the JB calendar, there was a captcha that needed to be completed and I couldn't complete it due the size of the iframe.
![cap](https://user-images.githubusercontent.com/6867792/208181209-850530cf-f3cb-4f5c-afa2-d70bbfccde5d.jpeg)

**Comment by CGBassPlayer** [comment]
So I tried both with and without my PIA VPN going to the Montreal server and mine loaded fine without a captcha. Not sure if that guest wifi is a flagged IP from Google?

**Comment by gerbrent** [comment]
@rastacalavera hmm!

Tricky to reproduce but if you have regular access to this network perhaps you can help us determine the correct size needed for the iframe?

**Comment by rastacalavera** [comment]
Yeah I could maybe do that. At a museum and it’s happening here too. Seems
like networks with a captive portal result in this behavior.

**Comment by esarver** [comment]
I got this captcha on my home Wi-Fi and had the same issue. I'm using Apple's "hide my ip" feature. Maybe that triggered it?

**Comment by CGBassPlayer** [comment]
I also have that turned on with my iPhone and I just got it for the first time on mobile. When this initial issue was opened, I could not get it to happen so I'm not sure what changed.

**Comment by rastacalavera** [comment]
Yeah maybe it has something to do with Apple private networking. I guess I had assumed that Google or whatever flags certain IP ranges so maybe the Apple private network is flagged and bigger things like universities or whatever also trigger it.

**Comment by reclaimingmytime** [comment]
I think this is a duplicate of issue #235. Also, there is still issue #243 regarding a replacement for Google Calendar.

I can't reproduce the issue with with the free version Browsec VPN though, so not all VPNs get flagged.
